.github/workflows/cron-job-templating/cron-template.j2

```diff
@@ -0,0 +1,177 @@
+name: cron-{{ input_epoch }}-core-{{ input_os }}-latest
+
+env:
+  os: {{ input_os }}-latest
+  distro: core
+  epoch: '{{ input_epoch }}'
```

Comment:
Once you re-implement the `jinja2.Environment` suggestions described elsewhere in this review you will need to update the variable tags here as necessary (and elsewhere in this template, but I'll only mark it here).